package.json

```diff
@@ -19,12 +19,12 @@
   "scripts": {
     "test": "yarn lint",
     "test:compiler": "jest ./tests",
-    "lint": "concurrently 'yarn lint:markdown' 'yarn lint:spelling' 'yarn lint:yaml' 'yarn lint:links'",
+    "lint": "concurrently 'yarn lint:links' 'yarn lint:markdown' 'yarn lint:spelling' 'yarn lint:yaml'",
```

Comment:
not a big deal, but this looks like a noop change?

Reply:
Agreed